### PR TITLE
Budget-App

### PR DESCRIPTION
**Pull request by lilyfrancis**
The Ruby on Rails capstone project ([remember what they are?](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/capstone_intro.md)) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category so that you can see how much money you spent and on what.

You will create a Ruby on Rails application that allows the user to:

register and log in, so that the data is private to them.
introduce new transactions associated with a category.
see the money spent on each category.
Use ruby gems as a software packages system.
Install Ruby on Rails framework.
Understand Rails RESTful design and router.
Use controllers to handle requests and render empty views.
Use params from browser requests in a safe way.
Use preprocessed HTML file with embedded Ruby code.
Use layouts and templates for shared content.
Use database migration files to maintain database schema.
Use validations for models.
Secure app from n+1 problems.
Understand what ORM is.
Write SQL queries with ActiveRecord.
Set up associations between models.
Build a web app that requires the user to log in.
Use devise gem for authentication.
Limit access to web app resources based on authorization rules.

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Copyright 2022, [MOHAMED ELSHAMY]
+
```
Comment (ahmetbozaci) [inline MIT.md]: - [ ] Please change this name with your name

File: app/views/expenditures/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <%end %>
+</div>
+
+<div id="expenditures">
+  <% @expenditures.each do |expenditure| %>
+    <%= render expenditure %>
+    <p>
+     
+    </p>
+  <% end %>
```
Comment (ahmetbozaci) [inline app/views/expenditures/index.html.erb]: - [ ] The list of payments should present ordered by the most recent. Please fix it.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @lilyfrancis  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Nice to add some flash messages. ✔️ 
- Deployed Application ✔️ 
- Nice video Presentation ✔️ 

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by rustam-tolipov — APPROVED** [review]
## Hello @lilyfrancis 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**